aspect/intellij_info_impl.bzl

```diff
@@ -414,6 +415,157 @@ def collect_go_info(target, ctx, semantics, ide_info, ide_info_file, output_grou
     update_sync_output_groups(output_groups, "intellij-resolve-go", depset(generated))
     return True
 
+def _crate_info(ctx):
```

Comment:
Each Rust target needs to know the following info about its dependencies:

- The crate name
- The crate version
- The crate's [features](https://doc.rust-lang.org/cargo/reference/features.html)

These three pieces of data are written directly into the Cargo manifest